lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -80,6 +80,11 @@ sub remove : Local Edit
     $c->model('ReleaseGroup')->load($release);
     $c->model('ReleaseGroup')->load_meta($release->release_group);
 
+    # For proper display of the Disc IDs tab
+    $c->model('Medium')->load_for_releases($release);
```

Comment:
How come the mediums need to be loaded? Doesn't the tab display only depend on `may_have_discids`?

Reply:
Never mind, answered my own question. 😆